experiments/configurations/qdrant-on-disk.json

```diff
@@ -11,5 +11,24 @@
       { "parallel": 8, "config": { "hnsw_ef": 128 } }
     ],
     "upload_params": { "parallel": 4 }
+  },
+  {
+    "name": "qdrant-all-on-disk-scalar-q",
+    "engine": "qdrant",
+    "connection_params": {},
+    "collection_params": {
+      "optimizers_config": { "default_segment_number": 17 },
+      "quantization_config": { "scalar": {"type": "int8", "quantile": 0.99, "always_ram": false} },
+      "vectors_config": { "on_disk": true },
+      "hnsw_config": { "on_disk": true, "m": 0, "payload_m": 16 },
+      "on_disk_payload": true
+    },
+    "payload_index_params": {
+      "on_disk": true
+    },
```

Comment:
You know what, we can do something like this:

```json
{
    "name": "qdrant-all-on-disk-scalar-q",
    "engine": "qdrant",
    "connection_params": {},
    "collection_params": {
      "optimizers_config": { "default_segment_number": 17 },
      "quantization_config": { "scalar": {"type": "int8", "quantile": 0.99, "always_ram": false} },
      "vectors_config": { "on_disk": true },
      "hnsw_config": { "on_disk": true, "m": 0, "payload_m": 16 },
      "on_disk_payload": true,
      "payload_index_params": {
        "a": { "is_tenant": true, "on_disk": true }
      }
    }
}
```

And throw error if the dataset doesn't have this field (`a`). So someone doesn't accidently run it on wrong dataset. It's valid because these are configurations and this gives more freedom.

Altering a dataset config (`dataset.json`) makes less semantic sense. So you can leave `datasets.json` just like it is.